SECURITY.md

```diff
@@ -0,0 +1,79 @@
+
+# Areas
+
+## Host security
+
+Plugins depend on use of the Java Security Manager is use to limit interactions on the host operation system resources (cpu/disk/memory/network/...).  JSM has been deprecated, with its removal scheduled in the next release of the JVM, see [OpenSearch discussion](https://github.com/opensearch-project/OpenSearch/issues/1687). Additional measures are needed to protect system resources.
+
+Extensions are sandboxed from the host system by operating via REST APIs.  This security boundary isolates extensions from executing operation system calls directly on OpenSearch hosts.
+
+## Communications security (COMSEC)
+
+Plugins are loaded into the same java virtual machine instance allowing communicate to OpenSearch through in process java APIs.  Plugins can issue REST API requests to the OpenSearch hosts reusing the standard node-to-node communications, internally called the transport client.
+
+Extensions of OpenSearch communicate via https requests between the nodes on the cluster and the extensions endpoint(s).  This is a bi-direction communication also allows extensions to contact the OpenSearch cluster through its available REST APIs.
+
+## Data Security
+
+OpenSearch stores data in memory and local file system storage.  This data is stored unencrypted.
+
+Plugins can use the existing data systems of the OpenSearch.  Several classes of plugins extend storage options out to external services.
+
+### Access Control
+
+With the security plugin installed, role based access control (RBAC) is available with a proprietary policy document format.  Access control over native OpenSearch data is possible with this plugin installed.
+
+For resource that are managed by plugins, access control is governed within individual plugin, by examining [user](https://github.com/opensearch-project/common-utils/blob/main/src/main/java/org/opensearch/commons/authuser/User.java) object from OpenSearch's thread context permissions are available for approval/denial. Example from anomaly detection, [checkUserPermissions](https://github.com/opensearch-project/anomaly-detection/blob/875b03c1c7596cb34d74fea285c28d949cfb0d19/src/main/java/org/opensearch/ad/util/ParseUtils.java#L568).  Uniform resource controls and models are needed to protect from misconfiguration and code defects.
+
+As Extensions do not have access OpenSearch's thread context, identity and its associated privileges must be communicated through the REST APIs.
+
+## Auditing
+
+With the security plugin installed, when actions are performed on the OpenSearch cluster they are recorded if filtering criteria are meet to configurable audit log sinks.
+
+## Installation
+
+Plugin installation is managed by using a binary on the node that extract plugin zip files into the file system, this is done outside the active running of OpenSearch itself.  When OpenSearch starts it loads installed plugins into its JVM runtime.
+
+Extensions installation is managed through on disk configuration. 
```

Comment:
I'm not clear on what this means (and specifically how it relates to security)

Reply:
This is relevant in that CVEs for plugins are tightly coupled with one another and OpenSearch.  Whereas extensions and OpenSearch are be able to perform minor/patch revisions for CVEs without coordination.  I've update the section to better capture this.